src/constants.rs

```diff
@@ -9,7 +9,9 @@ pub const SESSION_COOKIE_NAME: &str = "RSESSID";
 pub const CLIENT_URL_ENV_VAR: &str = "CLIENT_URL";
 pub const SESSION_LENGTH_SECONDS: i64 = 30 * 24 * 60 * 60; // 3 months
 pub const PASSWORD_SPECIAL_CHARS: &str = "!@#$%^&*";
+pub const LATEXMK_PATH: &str = "/bin/latexmk";
```

Comment:
in docker this doesn't work, change it to just "latexmk" or make it env variable

Reply:
Also, shouldn't we create the `/tmp/agar_service` directory in `Dockerfile`?

Reply:
You mean a volume? I assumed we don't need to persist the pdf files for now

Reply:
nevermind, I thought that if `/tmp/agar_service` doesn't work, the compilation will fail. I tested it and it's not true